src/common/Axis/LinearAxis/LinearAxisTickLabel.tsx

```diff
@@ -149,14 +154,15 @@ export const LinearAxisTickLabel: FC<Partial<LinearAxisTickLabelProps>> = ({
 
   const { x, y } = getOffset();
   const textPosition = getTextPosition();
+  const titleHover = formatTooltip ? formatTooltip(fullText) : fullText;
```

Comment:
You probably need to check if its a function ... typeof formatTooltip === function